js/package.json

```diff
@@ -36,12 +36,12 @@
     "ci:build:app": "NODE_ENV=production webpack --config webpack/app",
     "ci:build:lib": "NODE_ENV=production webpack --config webpack/libraries",
     "ci:build:dll": "NODE_ENV=production webpack --config webpack/vendor",
-    "ci:build:npm": "NODE_ENV=production webpack --config webpack/npm",
-    "ci:build:jsonrpc": "babel-node ./scripts/build-rpc-json.js --output .npmjs/jsonrpc",
+    "ci:build:npm": "NODE_ENV=production babel npm/$LIBRARY/src --out-dir npm/$LIBRARY/dist --ignore spec.js --source-maps",
+    "ci:build:jsonrpc": "babel-node ./scripts/build-rpc-json.js > npm/jsonrpc/index.json",
```

Comment:
Please no redirect. Do it in the script so it is obvious where things end up according to the actual source.